p/scripts/main.js

```diff
@@ -1511,6 +1540,23 @@ function parseJsonVars() {
 	window.icons = json.icons;
 }
 
+function init_dropdown_handler() {
```

Comment:
One thing that still needs to be done for sure is stick this in a more sensible place... I just kind of arbitrarily added it in before the actual initializations because I haven't really groked the organizational structure of main.js yet.

Reply:
I am not sure there is a great logic behind the current organisation 😄 
Things are supposed to be more or less grouped, trying also to put functions before their calls (minimising our dependency on [JavaScript hoisting](https://developer.mozilla.org/docs/Glossary/Hoisting)

Reply:
Ah, well that would certainly explain why I was having a hard time getting a feel for it, then!